config/locales/en.yml

```diff
@@ -2192,7 +2192,7 @@ en:
   label_yesterday: "yesterday"
   label_role_type: "Type"
   label_member_role: "Project Role"
-  label_global_role: "Global Role"
+  label_global_role: "Global role"
```

Comment:
The line above is `Project Role`. Why is this not changed as well?

Reply:
I for some reason recall having that committed in... Weird